┌───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
│ [debug] 9:26:17 764ms
│ This is a debug message
└───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
┌───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
│ [info] 9:26:17 769ms
│ Application started
└───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
┌───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
│ [warning] 9:26:17 770ms
│ This is a warning message
└───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
┌───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
│ [error] 9:26:17 770ms
│ This is an error message
│
│ Type: _Exception
│ Message: Exception: Test error
└───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
